<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1189</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Porte et Fenetre</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(&quot;C:/projet-new/projet/background1.jpg&quot;);background-size:cover;
background-repeat:no-repeat;</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1161</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Portes</string>
    </attribute>
    <widget class="QPushButton" name="ButtonExcel">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>370</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Excel</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>331</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Numéro de la porte :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Etat de la porte :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leDimensions">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>160</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Dimensions de la porte :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Description :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Matiére de la porte :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leMatiere">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>220</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spNumero">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QComboBox" name="cbEtat">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Ouverte</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fermee</string>
       </property>
      </item>
     </widget>
     <widget class="QTextEdit" name="textDescription">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>280</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="leRecherche">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>10</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>recherche</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonAjouterPorte">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSupprimerPorte">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>370</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QTableView" name="tabPorte">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>791</width>
       <height>321</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonModifererPorte">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>370</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifer</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbTri">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>10</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Tous</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Numéro</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Etat</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dimensions</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Matiére</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Déscription</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="buttonVideo">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>370</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Video</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonPDF">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>370</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Fenétre</string>
    </attribute>
    <widget class="QStackedWidget" name="stackedWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>1141</width>
       <height>531</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="afficherCarteFid">
      <widget class="QTableView" name="tabCarteFid">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>741</width>
         <height>361</height>
        </rect>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_recherche_carteFid">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>40</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>recherche</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_colone_carteFid">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>40</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Tous</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NUM_CARTE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DATE_DE_CREATION</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>POINTS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TOTAL_ACHATS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>STATUT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>REMISE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MATRICULE_FISCALE</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="button_ajouter_carteFid">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>470</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_supprimer_carteFid">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>470</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>470</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Retour</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="ajoutClient_2">
      <widget class="QDialogButtonBox" name="validerAjout_cf">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>500</y>
         <width>193</width>
         <height>28</height>
        </rect>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>651</width>
         <height>431</height>
        </rect>
       </property>
       <property name="title">
        <string>Ajout de carte de fidélité :</string>
       </property>
       <widget class="QSpinBox" name="sb_numCarte">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>50</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="sb_points">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>110</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Num carte :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Points :</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="le_dateCreation">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>50</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>50</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date_de_creation :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Total d'achats :</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="dsb_totalAchats">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>170</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>230</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Statut :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_25">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>290</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Remise :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_26">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>120</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Matricule fiscale :</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cb_Status">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>230</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Platinum</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Silver</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gold</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="cb_matriculeFiscale">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>120</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="sb_remise">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>290</y>
          <width>141</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="modifierClient_2">
      <widget class="QDialogButtonBox" name="validerModif_cf">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>500</y>
         <width>193</width>
         <height>28</height>
        </rect>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
      <widget class="QComboBox" name="cbTriFenetre">
       <property name="geometry">
        <rect>
         <x>900</x>
         <y>10</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Tous</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Numéro</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Etat</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dimensions</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Matiére</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Déscription</string>
        </property>
       </item>
      </widget>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>331</width>
         <height>351</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_43">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>111</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="text">
         <string>Numéro de la fenetre :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_44">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="text">
         <string>Etat de la fenetre :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="leDimensionsFenetre">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>160</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_45">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="text">
         <string>Dimensions de la fenetre :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_47">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>280</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="text">
         <string>Description :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_48">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="text">
         <string>Matiére de la fenetre :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="leMatiereFenetre">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>220</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spNumeroFenetre">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>50</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QComboBox" name="cbEtatFenetre">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>100</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Ouverte</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fermee</string>
         </property>
        </item>
       </widget>
       <widget class="QTextEdit" name="textDescriptionFenetre">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>280</y>
          <width>131</width>
          <height>61</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="buttonModifererFenetre">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>380</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonExcelFenetre">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>380</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Excel</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonSupprimerFenetre">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>380</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leRechercheFenetre">
       <property name="geometry">
        <rect>
         <x>1010</x>
         <y>10</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>recherche</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonAjouterFenetre">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>380</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QTableView" name="tabFenetre">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>40</y>
         <width>781</width>
         <height>331</height>
        </rect>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonPdfFenetre">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>380</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonVideoFenetre">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>380</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Video</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
